### PR TITLE
Update Minehut description in providers

### PR DESCRIPTION
**Pull request by GStudiosX2**
minehut now allows you to direct connect to servers via bedrock

### CODE REVIEW

**Review by onebeastchris — CHANGES_REQUESTED** [review]
Spelling of "or" should be capitalized; additionally, maybe shorten to:
"Or connect directly via [..]"

**Comment by Camotoy** [comment]
The exclamation point at the end is also too exciting of a tone for the wiki.

**Comment by GStudiosX2** [comment]
> The exclamation point at the end is also too exciting of a tone for the wiki.

alright i can remove that

**Comment by onebeastchris** [comment]
I'd still leave a dot at the end to indicate the end of the sentence

**Comment by GStudiosX2** [comment]
so any other changes I should make or is this good enough ?

**Review by onebeastchris — APPROVED** [review]
lgtm

File: _data/providers.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
     {
       "name": "Minehut",
       "url": "https://minehut.com/",
-      "description": "Connect via `bedrock.minehut.com` and do `/join <servername>`."
+      "description": "Connect via `bedrock.minehut.com` and do `/join <servername>`. Or connect directly via `<servername>.bedrock.minehut.gg`."
```
Comment (Chew) [inline _data/providers.json:54]:
```suggestion
      "description": "Connect via `bedrock.minehut.com` and do `/join <servername>`, or connect directly via `<servername>.bedrock.minehut.gg`."
```

**Review by Chew — APPROVED** [review]

**Pull request merged**